source/overloading.tex

```diff
@@ -3487,24 +3480,24 @@
 \indextext{overloading!unary operator}
 
 \pnum
-A prefix unary operator shall be implemented by a
-non-static member function\iref{class.mfct} with no parameters or a
-non-member function with one parameter.
+A \defnadj{prefix unary}{operator function}
+is a function named \tcode{\keyword{operator}@}
+for a prefix unary operator \tcode{@}\iref{expr.unary.op} that is either
+a non-static member function\iref{class.mfct} with no parameters or
+a non-member function with one parameter.
 \indextext{unary operator!interpretation of}%
-Thus, for any prefix unary operator
-\tcode{@},
-\tcode{@x}
-can be interpreted as either
-\tcode{x.op\-er\-a\-tor@()}
-or
-\tcode{operator@(x)}.
-If both forms of the operator function have been declared,
-the rules in~\ref{over.match.oper} determine which, if any, interpretation is
-used.
-See~\ref{over.inc} for an explanation of the postfix unary operators
-\tcode{++}
-and
-\tcode{\dcr}.
+For a \grammarterm{unary-expression}
+of the form \tcode{@ \grammarterm{cast-expression}},
+the operator function is selected by overload resolution\iref{over.match.oper}.
+If a member function is selected,
+the expression is interpreted as
+\tcode{\grammarterm{cast-expression} . \keyword{operator}@()};
+otherwise, the expression is interpreted as
```

Comment:
We also need to handle the case where a built-in candidate is selected.

"otherwise, if a non-member function is selected, the expression is interpreted as"

maybe?

Reply:
Sounds good.

I've also fixed the similar occurence for binary operators a few lines down.